### ISSUE TITLE
TNT Australia Provider response handling issues

### ISSUE THREAD

**Issue by zollf** [verbb/postie#94]
### Describe the bug

When using the provider "TNT Australia" and fetching shipping rates, the response returned from TNT Australia doesn't seem to be handled correctly for our case, it gives the valid response, but shows as an error. See:
![image](https://user-images.githubusercontent.com/63968092/197960407-d80fc483-3889-4e37-b692-1650b482799c.png)

When doing additional debugging i found that code around here could be a source of the problem potentially: https://github.com/verbb/postie/blob/9712169ee24e6c2d8f664cc6771f1c734f37d9e7/src/providers/TNTAustralia.php#L147-L152 

When printing out the XML response from TNT i get structure as follows
```xml
<?xml version="1.0"?>
<response>
  <ratedTransitTimeResponse>
    <ratedProducts>
      <ratedProduct>
        <product>
          <code>***</code>
          <description>***</description>
          <bookingCutOffTime>***</bookingCutOffTime>
          <collectionCutOffTime>***</collectionCutOffTime>
        </product>
        <quote currency="AUD">
          <price>***</price>
        </quote>
        <estimatedDeliveryDateTime>***</estimatedDeliveryDateTime>
      </ratedProduct>
      ...
    </ratedProducts>
  </ratedTransitTimeResponse>
</response>
```
then this gets parsed to `SimpleXMLElement` object. Type access seems to be object(->) not array ([...]). Whether this is the real source of the issue, i'm not 100% sure.

Expected behaviour is for a valid response to not be treated as an error. 

### Steps to reproduce

1. Enable and Configure TNT Australia Provider
2. Create an order
3. Fetch shipping rates

If a more sophisticate reproduction or a repository reproduction is needed, I'll be happy to create those.

### Craft CMS version

Craft Pro 3.7.58

### Plugin version

2.4.20

### Multi-site?

No

### Additional context

PHP Version: 7.4.30
OS version: Linux 5.10.124-linuxkit
Craft Commerce: 3.4.10.1

**Comment by engram-design** [comment]
Happy to look into this. Would you be able to share your login details (privately) for me to continue debugging this? Please send to `web at verbb.io` if you can.

But yes, it seems like from your findings we're not de-serializing the XML properly.

**Comment by engram-design** [comment]
Actually managed to find some login details (thanks to the kindness of @bossanova808) and fixed for the next release. To get this early, change your `verbb/postie` requirement in `composer.json` to:

```
"require": {
  "verbb/postie": "dev-craft-3 as 2.4.20",
  "...": "..."
}
```

Then run `composer update`.

**Comment by zollf** [comment]
@engram-design @bossanova808 thanks! 🎉

**Comment by engram-design** [comment]
Fixed in [2.4.21](https://github.com/verbb/postie/releases/tag/2.4.21)
